src/components/FetchRouteList.tsx

```diff
@@ -0,0 +1,70 @@
+import { Badge, Text } from "@fluentui/react-components";
+import { Card } from "@fluentui/react-components/unstable";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { RoutesXml } from "../data/etaXml";
+import RawDisplay from "./RawDisplay";
+import { FetchXMLWithCancelToken } from "./fetchUtils";
+
+const parseRouteTitle = (input: string) => {
+  const routeTitleRegex = /\d+-/;
+  if (routeTitleRegex.test(input)) {
+    return input.replace(routeTitleRegex, "");
+  } else {
+    return input;
+  }
+};
```

Comment:
can we do sth like 
```
const parseRouteTitle = (input: string) => {
  const routeTitleRegex = /\d+-/;
  return input.replace(routeTitleRegex, "");
  }
};
```

if regex will be used elsewhere, I suggest we should make it a generic fn and store it in utils folder

Reply:
seems unlikely for now